engine/deck.py

```diff
@@ -1,9 +1,15 @@
 from engine.card import Card
+from engine.color import Color
 from random import shuffle, Random
 
+
 class Deck(object):
 
-    def __init__(self, colors=('r', 'y', 'g', 'w', 'b'), numbers=(1, 1, 1, 2, 2, 3, 3, 4, 4, 5), seed=None):
+    def __init__(self,
+                 colors=(Color('red'), Color('yellow'), Color('green'), Color('white'), Color('blue')),
```

Comment:
Optional food for thought, because I'm undecided. A deck should not take in cards, it should create cards. As such, it seems a bit odd for colors (a component of cards) to be specified as instances of the Color class before cards are created.  My idea is basically "what should the gamestate object have to specify", and ideally we want to maximize readability and minimize required knowledge, and let it's constructor/container class constructors manage everything else.